pyquery/pyquery.py

```diff
@@ -12,6 +12,7 @@
 import types
 import sys
 
+from collections import OrderedDict
```

Comment:
Since Python 3.7: Dictionary order for all dicts is guaranteed to be insertion order.